corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -393,16 +399,19 @@ FormplayerFrontend.on('refreshApplication', function(appId) {
     if (!appId) {
         throw new Error('Attempt to refresh application for null appId');
     }
+    var sessionId = FormplayerFrontend.request('getCurrentSessionId');
```

Comment:
In that case the sessionId will be null on Formplayer too and we'll just do a normal update with no attempt at replaying anything